lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('@sumup/foundry/lint-staged')({
```

Comment:
Are you going to also to the `postinstall` approach here?

Reply:
This is the config for the wrapper project, not the template. It enables linting of the template files without initializing Foundry inside the template folder. The template itself already uses the `postinstall` approach.

Reply:
🤦 I was tired. 😆